test/runtests.jl

```diff
@@ -3,6 +3,7 @@ include("front_matter.jl")
 @testset "Mooncake.jl" begin
     if test_group == "aqua"
```

Comment:
We should probably rename this test group to "automated quality assurance", or really anything which highlights that it's not just Aqua.jl anymore.

Reply:
done